core/arch/riscv/kernel/boot.c

```diff
@@ -120,7 +120,7 @@ __weak void primary_init_intc(void)
 }
 
 /* May be overridden in plat-$(PLATFORM)/main.c */
-__weak void secondary_init_intc(void)
+__weak void main_secondary_init_intc(void)
```

Comment:
This name does not make much sense IMO, what is the main word supposed to represent? I'd rather do the opposite replacement in all Arm platforms.

Reply:
It is originated from traditional ARM code

I can see @maroueneboubakri  just rename `main_secondary_init_gic()` to `main_secondary_init_intc()` in commit https://github.com/OP-TEE/optee_os/pull/5718/commits/7cfd2b24191d9fc63993c571b6f21401656580fc

Reply:
That's commit https://github.com/OP-TEE/optee_os/commit/ef50391ee92b850d6a6f9280969ed4a9857480af in the master branch actually. Well I wasn't careful enough when reviewing, because I agreed with the description which states "Rename secondary_init_gic() to secondary_init_intc()", but it is obviously not what the code does.

I still believe `main_secondary_*()` is a bad name unless someone explains what is it supposed to mean.

Reply:
Find a commit 7 years ago https://github.com/OP-TEE/optee_os/commit/05efe1e1222d69cfbb68a1aeca9a9af2c310a7aa

Reply:
Resolved in https://github.com/OP-TEE/optee_os/pull/6207